### PR TITLE
Compress Website Sass output

### PR DESCRIPTION
**Pull request by DirtyF**
Suggested by @pathawks in #6008

### CODE REVIEW

File: rake/site.rake [hunk 1]
```diff
@@ -47,7 +47,7 @@ namespace :site do
   task :update_normalize_css do
     Dir.chdir("#{docs_folder}/_sass") do
       sh 'curl "https://necolas.github.io/normalize.css/latest/normalize.css" -o "normalize.scss"'
-      sh 'sass "normalize.scss":"_normalize.scss" --style compressed'
+      sh 'sass "normalize.scss":"_normalize.scss"'
```
Comment (pathawks) [inline rake/site.rake]: What is the purpose of this?

Why not:

```sh
sh 'curl "https://necolas.github.io/normalize.css/latest/normalize.css" -o "_normalize.scss"'
```

Comment (DirtyF) [inline rake/site.rake]: It seems that previously we used a Sass workflow, and now we rely entirely on GitHub Pages pipeline. We therefore can simplify here to the bare minimum, you're right. 

Suggestion applied. 🖖

**Comment by pathawks** [comment]
…as long as there isn't some reason I'm not thinking of to _not_ do this

**Comment by parkr** [comment]
This is automatically done in production - are you seeing different results on jekyllrb.com?

**Comment by DirtyF** [comment]
@parkr https://jekyllrb.com/css/screen.css does not look compressed to me

**Comment by parkr** [comment]
You're absolutely right! It defaults to `:compact`, which isn't the same.

**Comment by parkr** [comment]
@jekyllbot: merge +docs

**Pull request merged**